test/Microsoft.Extensions.Logging.Test/ConsoleLoggerTest.cs

```diff
@@ -63,16 +63,19 @@ public void LogsWhenMessageIsNotProvided()
             var sink = t.Item2;
             var exception = new InvalidOperationException("Invalid value");
 
+            ((TestConsole)t.Item1.Console).OnFlush = () =>
```

Comment:
Isn't there technically a race here, where not all `Log*` were called before `OnFlush` is called?